sensor/kubernetes/sensor/sensor.go

```diff
@@ -101,9 +101,12 @@ func CreateSensor(cfg *CreateOptions) (*sensor.Sensor, error) {
 		return nil, errors.Wrap(err, "creating enforcer")
 	}
 
+	// TODO: Move other singleton stores into the store provider
```

Comment:
You got a ticket ref for this? Otherwise this todo would be easy to forget

Reply:
+1